web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
+- Navigating to the metrics page presents users with an empty query field and a prompt to enter a query in the box
+
+![Metrics page with too many results to graph](img/metrics-full.png)
+- If a query cannot be graphed, the results should still be shown in the table below, but the graph field will be populated by an error message until the issue is resolved or the offending query is switched off.
+
+![Metrics page with un-graphable results](img/metrics-error.png)
+- In some cases, we could make specific suggestions about how to improve a given query such as by adding a function
+
+![Query field with autocomplete](img/metrics-autocomplete.png)
+- As the user types, matching metrics and functions appear in a list below. Selecting or clicking on one will fill it into the query field
+
+![Query field with specific autocomplete result](img/metrics-autocomplete-2.png)
+- Results are narrowed as the entered text gets more specific
+
+![Metrics page with a graph of results](img/metrics-results.png)
+- When a valid query is entered, results are graphed and displayed in a table below the query.
+
+![Metrics graph with a hidden series](img/metrics-hide-series.png)
+- Clicking on a result row will toggle the visibility of that series in the graph. Hidden rows will have their legend color block replaced by an outline until they are turned back on by clicking the row again
+
+![Metrics graph with two different queries](img/metrics-add-query.png)
+- Multiple queries can be graphed at the same time by using the “Add query” action.
+- Clicking on the caret by a query will hide and show the table associated with the query
+- The option to collapse all query tables is available in the actions menu to help reduce noise from large result sets
+
+
+![Metrics graph with one query turned off](img/metrics-hide-query.png)
```

Comment:
Is there a hover to explain this to the user? I wasn't able to guess what this was, but it of course makes complete sense after you explained it here :)

Reply:
I wonder if show/hide might be clearer than on/off?

Reply:
@ncameronbritt are you thinking those labels within the switch, or some kind of button? I think it's important that we clearly display the state of each query clearly and the switch is a simple way to do that

Reply:
Yes, within the switch. Was just a thought based on @lizsurette's comment.

Reply:
Had the same thought. Show/Hide would be clearer, but the switch would get pretty large. Do we have a pattern/component for an Eye icon and a slashed variant that could be toggled, with a label that switches between the words "Show" and "Hide"?

Reply:
+1 to Show/Hide!

Reply:
I've updated the notation here and added in an explicit tooltip (especially useful because the PF4 switch has no label). I think the "show/hide" option is interesting, but given what this action does, I think the "on/off" switch is a better metaphor